app/controllers/merchant/dashboard_controller.rb

```diff
@@ -1,4 +1,11 @@
 class Merchant::DashboardController < Merchant::BaseController
   def index
+    merchant_id = MerchantUser.where(user_id: params[:id]).pluck(:merchant_id).pop
```

Comment:
We should refactor 'merchant_id.nil? == false' into something that looks a bit better.